basics_activity/_basics.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Learn Cadet Basics"
+description: "Learn what a database and SQl are"
+date: 2020-07-26
+prereq: "none"
+difficulty: "Beginner"
+icon: ""
+draft: false
+translationKey: "sql_basics"
+---
+# Cadet Training!
+<font size="4"> As a new honorary space explorer, let's go over some basics!
+
+## What Is A Database?
+
+<font size="3"> It is a type of container for you to hold and organize information. 
+Think of a database like a bucket where you can place toys in so they don’t get lost 
+and you can easily find them later. This bucket might hold smaller buckets to sort 
+your toys.
+
+![Analogy](../assets/Database_Analogy.png)

```

Comment:
Do we own these pictures? If not, make sure we have added credits :)

Reply:
I saw the pictures, looks like we created them from scratch! 💯

Reply:
I used Google Drawings to create the pictures, do I need to cite any software I used to draw the pictures? Maybe I can add a small comment at the bottom of the page? Something like "Images designed with Google Drawings"?